modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/CKEditorBalloonEditorConfigContributor.java

```diff
@@ -125,7 +125,7 @@ protected JSONArray getStyleFormatsJSONArray(Locale locale) {
 	protected String getToolbarText() {
 		return "Styles,Bold,Italic,Underline,BulletedList,NumberedList," +
 			"TextLink,JustifyLeft,JustifyCenter,JustifyRight,JustifyBlock," +
-				"BGColor,RemoveFormat";
+				"BGColor,RemoveFormat,LineSpacing";
```

Comment:
Don't forget to keep the default options in the [BallonEditor](https://github.com/liferay/liferay-portal/blob/d0f6b0ff617f356c63894822b27b21f09a39fdb3/modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/BalloonEditor.js#L29-L35) (React) component "syncrhonized" with this `EditorConfigContributor`

Reply:
:joy:

Reply:
Added! Thanks ~god~! Oops, @julien!